### ISSUE TITLE
Support 10bits

### ISSUE THREAD

**Issue by SnAFKe** [pinterf/FFT3dGPU#1]
This plugin not support 10bits with FFT3dFilter work fine.

**Comment by hyllm** [comment]
yeah... but fft3dfilter is slow.... limits encode speed to around 2.5fps with 4K source denoising... without fft3dfilter it runs 10.1fps

**Comment by pinterf** [comment]
New release, 0.8.6. Tested on exactly one machine.
Pls check 1.) if works 2.) speed (both 10 and 16 bits, maybe float)
